packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.test.ts

```diff
@@ -170,6 +170,84 @@ describe('GraphQLSitemapService', () => {
       ]);
     });
 
+    it('should return personalized paths when personalize data is returned', async () => {
+      const lang1 = 'ua';
+
+      nock(endpoint)
+        .post('/')
+        .reply(200, {
+          data: {
```

Comment:
Can this data be added to the `testData/sitemapServiceResult`?

Reply:
done - added two json outputs based on personalize - non-personalize usage